src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -34493,6 +34493,55 @@ public class C
 ");
             Assert.Null(model.GetOperation(node3).Parent);
         }
+
+        [Fact]
+        public void OutVarInConstructorUsedInObjectInitializer()
+        {
+            var source =
+@"
+public class C
+{
+    public int Number { get; set; }
+    
+    public C(out int n)
+    {
+        n = 1;
+    }
+
+    public static void Main()
+    {
+        C c = new C(out var i) { Number = i };
+        System.Console.WriteLine(c.Number);
+    }
+}
+";
+            var compilation = CreateCompilation(source, options: TestOptions.DebugExe, parseOptions: TestOptions.Regular);
+            CompileAndVerify(compilation, expectedOutput: @"1");
```

Comment:
Nit: you could just use `CompileAndVerify`. Not blocking.

Reply:
Thanks, this was just copy/paste from the other tests. Cleaned up.